images/recycler/scripts/recycler.sh

```diff
@@ -22,8 +21,10 @@ if [[ ! -d ${dir} ]]; then
     exit 1
 fi
 
-# shred all files
-find ${dir} -type f -exec shred -fuvz {} \;
+find ${dir} -type f -exec stat -c "%u %g %n"  {} \; |xargs -n 3 bash -c 'sudo -u "#$1" shred  $3' argv0
```

Comment:
> uid 0 will be squashed if root_squash is turned on. 

so what will `sudo find` do that `find` as uid 0 would not? just trying to understand how it helps us